uint8 object_id
string object_type
float64 loc_x
float64 loc_y
float64 vel
float64 accel
